tests/lightning_tensor/test_var.py

```diff
@@ -0,0 +1,407 @@
+    )
+    def test_custom_wires(self, theta, phi, wires):
+        """Tests custom wires."""
+        device = LightningTensor(wires=wires, c_dtype=np.complex128)
+
+        tape = qml.tape.QuantumScript(
+            [
+                qml.RX(theta, wires=wires[0]),
+                qml.RX(phi, wires=wires[1]),
+                qml.CNOT(wires=wires),
+            ],
+            [qml.var(qml.PauliZ(wires=wires[0])), qml.var(qml.PauliZ(wires=wires[1]))],
+        )
+
+        calculated_val = execute(device, tape)
+        reference_val = np.array(
+            [1 - np.cos(theta) ** 2, 1 - np.cos(theta) ** 2 * np.cos(phi) ** 2]
+        )
+
+        tol = 1e-5 if device.c_dtype == np.complex64 else 1e-7
+        assert np.allclose(calculated_val, reference_val, atol=tol, rtol=0)
+
+    @pytest.mark.parametrize(
+        "Obs, Op, expected_fn",
+        [
+            (
+                [qml.PauliX(wires=[0]), qml.PauliX(wires=[1])],
+                qml.RY,
+                lambda theta, phi: np.array(
+                    [1 - np.sin(theta) ** 2 * np.sin(phi) ** 2, 1 - np.sin(phi) ** 2]
+                ),
+            ),
+            (
+                [qml.PauliY(wires=[0]), qml.PauliY(wires=[1])],
+                qml.RX,
+                lambda theta, phi: np.array([1, 1 - np.cos(theta) ** 2 * np.sin(phi) ** 2]),
+            ),
+            (
+                [qml.PauliZ(wires=[0]), qml.PauliZ(wires=[1])],
+                qml.RX,
+                lambda theta, phi: np.array(
+                    [1 - np.cos(theta) ** 2, 1 - np.cos(theta) ** 2 * np.cos(phi) ** 2]
+                ),
+            ),
+            (
+                [qml.Hadamard(wires=[0]), qml.Hadamard(wires=[1])],
+                qml.RY,
+                lambda theta, phi: np.array(
+                    [
+                        1 - (np.sin(theta) * np.sin(phi) + np.cos(theta)) ** 2 / 2,
+                        1 - (np.cos(theta) * np.cos(phi) + np.sin(phi)) ** 2 / 2,
+                    ]
+                ),
+            ),
+        ],
+    )
+    def test_single_wire_observables_variance(self, Obs, Op, expected_fn, theta, phi, dev):
+        """Test that variance values for single wire observables are correct"""
+
+        tape = qml.tape.QuantumScript(
+            [Op(theta, wires=[0]), Op(phi, wires=[1]), qml.CNOT(wires=[0, 1])],
+            [qml.var(Obs[0]), qml.var(Obs[1])],
+        )
+        result = execute(dev, tape)
+        expected = expected_fn(theta, phi)
+
+        tol = 1e-5 if dev.c_dtype == np.complex64 else 1e-7
+        assert np.allclose(result, expected, atol=tol, rtol=0)
+
+    def test_hermitian_variance(self, theta, phi, dev):
+        """Tests an Hermitian operator."""
+
+        with qml.tape.QuantumTape() as tape:
+            qml.RX(theta, wires=0)
+            qml.RX(phi, wires=1)
+            qml.RX(theta + phi, wires=2)
+
+            for idx in range(3):
+                qml.var(qml.Hermitian([[1, 0], [0, -1]], wires=[idx]))
```

Comment:
similar question: should we add tests for Hermitian with more than one wire, targeting at non-local sites?

Reply:
Same as above :+1: